<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHOOSE</class>
 <widget class="QDialog" name="CHOOSE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工艺选项区</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="pushButton_37">
       <property name="text">
        <string>毛坯正火态</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_38">
       <property name="text">
        <string>渗碳</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_39">
       <property name="text">
        <string>淬火</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_40">
       <property name="text">
        <string>回火</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="pushButton_34">
     <property name="text">
      <string>保存模板</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>毛坯</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>毛坯调制态</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_32">
         <property name="text">
          <string>渗碳淬火热处理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_33">
         <property name="text">
          <string>表面淬火</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_36">
     <property name="text">
      <string>生成工序表</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_35">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>工艺显示生成区</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>热处理</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="6" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>车削</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rough_turning_2">
         <property name="font">
          <font>
           <family>.AppleSystemUIFont</family>
          </font>
         </property>
         <property name="text">
          <string>粗车</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_41">
         <property name="text">
          <string>半精车</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_42">
         <property name="text">
          <string>镗削</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_43">
         <property name="text">
          <string>车/镗孔</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>铣削</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_44">
         <property name="text">
          <string>粗铣</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_45">
         <property name="text">
          <string>半精铣</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_46">
         <property name="text">
          <string>插孔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_47">
         <property name="text">
          <string>插键槽</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>磨削</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_48">
         <property name="text">
          <string>粗磨</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_49">
         <property name="text">
          <string>精磨</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_50">
         <property name="text">
          <string>拉孔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_51">
         <property name="text">
          <string>拉键槽</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="pushButton_52">
         <property name="text">
          <string>插齿</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_53">
         <property name="text">
          <string>滚齿</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_54">
         <property name="text">
          <string>铣齿</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_55">
         <property name="text">
          <string>镀铜</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_56">
         <property name="text">
          <string>消应力</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="pushButton_57">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_58">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_59">
         <property name="text">
          <string>磨齿</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_60">
           <property name="text">
            <string>喷丸</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_61">
           <property name="text">
            <string>深冷</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_62">
           <property name="text">
            <string>冰冷</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_63">
           <property name="text">
            <string>自定义</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>删除工序</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
